CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# dbt_netsuite_source v0.11.0
+[PR #57](https://github.com/fivetran/dbt_netsuite_source/pull/57) includes the following update: 
+
+## Breaking Changes 
+- Casted specific timestamp fields across all staging models as dates where the Netsuite UI does not perform timezone conversion. Keeping these fields as type timestamp causes issues in reporting tools that perform automatic timezone conversion. 
+  - As this will change the datatype of the underlying fields, this will require a  `--full-refresh`.
+- Existing fields that were converted from timestamp to date in the following `stg_netsuite2__*` models:
+  - `accounting_periods`: `starting_at` and `ending_at`
+  - `customers`: `date_first_order_at`
+  - `transactions`: `transaction_date`, `closed_at` and `due_date_at`
+- Adds additional commonly used fields within the `stg_netsuite2__*` models.
+  - `accounts`: `display_name`
+  - `customers` and `vendors`: `alt_name` 
+  - `subsidiaries`: `is_elimination`
+  - `transaction_accounting_lines`: `exchange_rate`
+  - `transaction_lines`: `is_eliminate`, `netamount`
+  - `transactions`: `reversal_transaction_id`, `reversal_date`, `is_reversal_defer`
+- **IMPORTANT**: Nearly all of these models have pass-through functionality. So you will need to remove these fields from your passthrough variable setup if they are currently present to avoid errors.
+
+## Feature Updates
+- Introduced the `stg_netsuite2__employees` model to bring in data from the `employee` source table. This was brought in to leverage fields like `first_name`, `last_name` and `supervisor` in downstream models in the `dbt_netsuite` transformation package. 
+  - Since this model is only used by a subset of customers, we've introduced the variable `netsuite2__using_employees` to allow users who don't utilize the `employee` table in Netsuite2 the ability to disable that functionality within your `dbt_project.yml`.
```

Comment:
Can you link to the README section to show users _how_ to disable if they are interested.

Reply:
It actually looks like this wasn't documented in the README. Would you be able to include this in the same section as the other disable/enable variables.

Reply:
Well that's annoying. Didn't save the file so it didn't commit. It's in now.